dockers/hypervisor/Dockerfile

```diff
@@ -1,45 +1,53 @@
-FROM isard/alpine-pandas:1.0.0
+FROM alpine:3.9
 MAINTAINER isard <info@isard.com>
 
-RUN pip3 uninstall pandas pytz python-dateutil six -y
+RUN apk add --no-cache \
+    qemu-system-x86_64 \
+    libvirt \
+    netcat-openbsd \
+    libvirt-daemon \
+    dbus \
+    polkit \
+    qemu-img \
+    openssh \
+    curl \
+    python3 \
+    supervisor
+
+RUN apk add --no-cache --virtual .build_deps \
+    build-base \
+    python3-dev 
+RUN pip3 install --no-cache-dir websockify==0.8.0
+RUN apk del .build_deps
 
-RUN apk --no-cache add qemu-system-x86_64 libvirt netcat-openbsd libvirt-daemon dbus polkit qemu-img
 RUN ln -s /usr/bin/qemu-system-x86_64 /usr/bin/qemu-kvm
-RUN apk add openssh curl bash
-RUN ssh-keygen -A
-ADD dockers/hypervisor/reset-hyper.sh /
-RUN chmod 744 reset-hyper.sh
 
+# SSH configuration
 RUN echo "root:isard" |chpasswd
```

Comment:
I'm not sure if this line is required. We should test it

Reply:
We leave this as it could be useful in development target